### DISCUSSION TITLE
Custom Summary on history json

### DISCUSSION THREAD

**Discussion by Sakretsos** [upptime/upptime#709] [Ideas]
Hello, hope you are all well!

I am new on this community and really i want to thank all the developers for this awesome work.
I dont know if this idea is already on the app but i think is a awesome for peoples who want a custom summary json.

My idea is the possibility peoples to configurate the summary file on .upptimerc.yml.
So if i need to hide something without making a new wrapper file this can be more helpful.
Some applications have characters limits, so if i add more than 2 monitors on a third party applications this cannot work and some characters missing.

Thanks and sorry for my english since is not my native language.

**Comment by AnandChowdhary** [comment]
Thank you for your message! What's the use case you have in mind?

**Reply by Sakretsos** [reply]
Hello @AnandChowdhary,

Thanks for your fast response!

Well, i use the awesome upptime monitor for my servers and i have write a script for a virtual reality world.
The problem there that the scripts have a limit on maximum (UTF-8 encoded) byte length of the HTTP response body.
Reference: https://wiki.secondlife.com/wiki/HTTP_BODY_MAXLENGTH.

Bellow you can find a screenshot from inworld body response when i use 3 servers for monitor.
![screenshot](https://user-images.githubusercontent.com/8127121/212189184-79af7c24-ed10-452f-a96d-48efd2a07caf.png)

As you can see on the screenshot some characters missing, this caused because of script limit.
So i think if peoples can modify the summary.json will help from this kind of problems.

Thanks,
Ioannis
